lib/python/qmk/cli/fileformat.py

```diff
@@ -0,0 +1,13 @@
+"""Format files according to QMK's style.
+"""
+from milc import cli
+
+import subprocess
+
+
+@cli.subcommand("Format files according to QMK's style.", hidden=True)
+def fileformat(cli):
+    """Run several general formatting commands.
```

Comment:
Could we also ensure files end in exactly one newline here? I think cformat does this but that obviously only works for .c and .h files.

Reply:
Im sure we could, I was just trying to cover the existing use cases. Can we assume all files in this context? or do we filter it to something like known code file extensions?

Reply:
Going to tackle newline as a separate PR, as this allows us to move of travis with the existing functionality.